test/unit/services/test-get-sub-nav.js

```diff
@@ -19,27 +25,30 @@ describe('services/get-sub-nav', function () {
   })
 
   it('returns the correct titles', function () {
-    var subNav = getSubNav(id, organisationalUnitName)
```

Comment:
Worth also verifying that the four returned for the other org units are the four we want

Reply:
done